docs/python/tutorials/integrating-dbos.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 10
+title: Adding DBOS To Your App
+pagination_next: null
+---
+import InstallNode from '/docs/partials/_install_node.mdx';
+import LocalPostgres from '/docs/partials/_local_postgres.mdx';
+
+### Using DBOS Transact
+
+This guide shows you how to add the open source [DBOS Transact](https://github.com/dbos-inc/dbos-transact-py) library to your application to **durably execute** it and make it resilient to any failure.
+
+#### 1. Install DBOS
+<section className="row list">
+<article className="col col--6">
+`pip install` DBOS into your application, then create a DBOS configuration file.
+</article>
+
+<article className="col col--6">
+
+```shell
+pip install dbos
+dbos init --config
+```
+
+</article>
+</section>
+
+#### 2. Add the DBOS Initializer
+<section className="row list">
+<article className="col col--6">
+
+In your program's main function, add these two lines of code.
+These initialize DBOS when your program starts.
+
+</article>
+
+<article className="col col--6">
+
+```python
+DBOS()
+DBOS.launch()
+```
```

Comment:
This would deserve a link to the [class reference](https://docs.dbos.dev/python/reference/dbos-class). Also it would make sense to give pointers about a fastapi or flask integration.